catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStorePublisher.java

```diff
@@ -210,7 +210,7 @@ public void handleEvent(Event event) {
                 (RegistryStore) bundleContext.getService(((ServiceEvent) event.getProperty(
                         EventConstants.EVENT)).getServiceReference());
 
-        if (!previousAutoPush && registryStore.isAutoPush()) {
+        if (registryStore.isAutoPush()) {
```

Comment:
✏️  We should also be able to remove the `previousAutoPush` variable and change lines L199-L201 to:
```
if (registryStoreMap.get(pid)) {
    return;
}
```